src/tests/fixtures/custom_scalars/DefineCustomScalar.ts.expected

```diff
@@ -18,10 +18,21 @@ type SomeType {
   hello: String
 }
 
-scalar MyUrl
+scalar MyUrl @exported(tsModulePath: "grats/src/tests/fixtures/custom_scalars/DefineCustomScalar.ts", functionName: "MyUrl", argCount: 0)
```

Comment:
TODO: Rename `functionName` to something more generic.
TODO: Make `argCount` optional?